site/data/qasida/ya-nasim-al-ward.json

```
@@ -5,34 +5,61 @@
     },
     "verses": {
         "ara": [
-            ["يَا نَسِيمَ الوَرْدِ خَبِّرْ لِلرَشَا"],
-            ["لَم يَزِدْنِي الوِرْدُ إلَا عَطَشَا"],
+            ["لَا إِلَهَ إِلاَّ اللهْ اللهْ اللهْ"],
+            ["لَا إِلَهَ إِلَّا اللَّهُ جُدْ عَلَيْنَا"],
```

```suggestion
            ["لَا إِلَهَ إِلَّا اللهْ جُدْ عَلَيْنَا"],
```
(Changed to sukoon to match Songs of Presence)